docs/source/starter/style_guide.rst

```diff
@@ -49,7 +51,7 @@ Here's a LightningModule that defines a system:
 .. testcode::
 
     class LitModel(LightningModule):
-        def __init__(self, encoder: nn.Module = None, decoder: nn.Module = None):
```

Comment:
n00b question: why remove the typehint?

Reply:
we don't usually have typehints in doc examples.

Reply:
yes agreed